kokoro/scripts/linux/build-docker.sh

```diff
@@ -69,6 +69,11 @@ if [ $TOOL = "cmake" ]; then
   if [ $CONFIG = "ASAN" ]; then
     ADDITIONAL_CMAKE_FLAGS="-DSPIRV_USE_SANITIZER=address,bounds,null"
     [ $COMPILER = "clang" ] || { echo "$CONFIG requires clang"; exit 1; }
+  elif [ $CONFIG = "UBSAN" ]; then
+    # UBSan requires RTTI, and by default UBSan does not exit when errors are
+    # encountered - additional compiler options are required to force this.
+    ADDITIONAL_CMAKE_FLAGS="-DSPIRV_USE_SANITIZER=undefined -DENABLE_RTTI=ON -DCMAKE_C_FLAGS=-fno-sanitize-recover -DCMAKE_CXX_FLAGS=-fno-sanitize-recover"
```

Comment:
Bit confused what `SPIRV_USE_SANITIZER=undefined` is all about. A comment would be good.

Reply:
It's a SPIR-V Tools CMake option for enabling UBSan (it causes -fsanitize=undefined to get used during compilation).

I have added a comment to clarify.